tutorials/containers/deploying-storageos-on-kubernetes/deploying-storageos-on-kubernetes.md

```diff
@@ -0,0 +1,333 @@
+---
+id: deploying-storageos-on-kubernetes
+summary: Learn how to deploy StorageOS on Charmed Kubernetes
+categories: containers, kubernetes, csi, storageos
+tags: juju, kubernetes, beginner, container, storageos, cloud
+difficulty: 3
+status: draft
+published: 2019-09-16
+author: Peter De Sousa <peter.de.sousa@canonical.com>
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+
+---
+
+# Deploying StorageOS on Charmed Kubernetes
+
+## Overview
+Duration: 5
+
+Kubernetes offers a range of storage solutions out of the box but the majority of these are specific to cloud providers, for example gcp or aws, this means that the options left for baremetal deployments is Ceph, localhost or NFS.
+
+StorageOS is a newcomer to this arena providing an easy to setup solution for storage in Charmed Kubernetes which is up and running within minutes.
+
+### In this tutorial you'll learn how to...
+
+- Deploy the StorageOS operator
+- Create a secret for StorageOS
+
+
+### You will only need
+
+- Ubuntu with MicroK8s
+- a Charmed Kubernetes deployment
+- or Multipass [for Windows](6https://ubuntu.com/blog/kubernetes-on-windows-how-to-set-up) or [Mac](https://tutorials.ubuntu.com/tutorial/install-microk8s-on-mac-os#0) if using [MicroK8s](https://tutorials.ubuntu.com/tutorial/install-a-local-kubernetes-with-microk8s#0)
+
+
+## Installing StorageOS
```

Comment:
Are you really "Installing StorageOS" in this step?

Reply:
Well it technically a deployment, the term 'install' comes from the StorageOS docs, but I guess it's a little misleading in this context